### PR TITLE
Simple Practice of For Loops

### PR DESCRIPTION
**Pull request by kalez43**
Three small for loops

### CODE REVIEW

File: for_loop_practice.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+questions = ["What is 1 + 1?", "What is 2 + 2?"]
+answers = [2, 4]
+
+for question in questions:
+    print(question)
+    user_answer = input("Type answer and press enter ==> ")
+    user_answer = int(user_answer)
+    if user_answer == int(answers[0]) or user_answer == answers[1]:
+        print("Well done!")
```
Comment (Bazarovay) [inline for_loop_practice.py]: You should check if the answer is equal to the answer of the correct question. Say I add 5 more questions, I would need to update this condition. You could do something like this
```
for i in range(0, len(questions)): # len(questions) will be the number of questions
    current_question = questions[i]
    current_answer = answers[i]
    # and then you can compare current answer and user answer
```

File: for_loop_practice.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+letters = "three"
+numbers = [0, 1, 2, letters, 4, 5, 6]
+
+for num in numbers:
+    print(num)
+
+print('***')
+
+colors = ["green", "blue", "pink", "yellow", "orange"]
+
```
Comment (Bazarovay) [inline for_loop_practice.py]: Please add comments about what the function is doing. Also, move them into functions. You should make it a habit

File: for_loop_practice.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+questions = ["What is 1 + 1?", "What is 2 + 2?"]
```
Comment (Bazarovay) [inline for_loop_practice.py]: There is no file description

File: for_loop_practice.py (hunk 1 above)
Comment (kalez43) [reply for_loop_practice.py]: I like that

**Pull request closed without merging**